### DISCUSSION TITLE
wo finde ich die beschriebenen .bin Dateien ?

### DISCUSSION THREAD

**Discussion by ThomasKlaus-ai** [tbnobody/OpenDTU#277] [General]
Guten Abend an die Runde,
in der Beschreibung steht unter dem Tab Actions soll man die bin Dateien finden, die man für den Flash eines ESP32 benötigt. Leider finde ich unter der beschriebenen Seite keine bin Dateien. Kann mir Jemand helfen ?
Viele Grüße
Thomas

**Comment by ghost** [comment]
Hier
https://github.com/tbnobody/OpenDTU/discussions/252
sollten die notwendigen Informationen zu finden sein.

**Comment by ThomasKlaus-ai** [comment]
Vielen Dank und ich habe es damit geschafft meinen ESP32 zu flashen. Leider bildet er nach dem Reset keinen Access Point und ich kann ihm damit auch keine Netzwerkeinstellungen geben. Liegt es an meinem PC und an irgendwelchen Sicherheitseinstellungen ?

**Comment by ghost** [comment]
Ob es an deimen PC liegt ist für mich von der Ferne schwer zu beurteilen. Bei mir hatte es mit dem PC auch schon gehakt, mit dem Smartphone klappte es dann problemlos, also
- unter WLAN den Access Point "OpenDTU-*" auswählen,
- dann Schritt für Schritt weiter wie hier https://github.com/tbnobody/OpenDTU#first-configuration
beschrieben.

**Comment by ThomasKlaus-ai** [comment]
Leider funktioniert es auch mit dem smartphone nicht. In meinem Router sehe ich den ESP32, aber er bekommt keinen Zugriff und im WLAN sehe ich keinen Access Point. Kann es sein, daß nicht alle ESP32 mit der software funktionieren ?

**Comment by jab42** [comment]
infos zum flashen:
https://github.com/lumapu/ahoy/wiki/FAQ-Frequently-Asked-Questions
oder etwas übersichtlicher:
https://discord.com/channels/984173303147155506/1030622476696309800
https://discord.gg/WzhxEY62mB
